etna/collections/models.py

```diff
@@ -124,6 +126,16 @@ def results_pages(self):
     ]
     subpage_types = ["collections.TopicExplorerPage", "collections.ResultsPage"]
 
+    def get_context(self, request):
```

Comment:
@mozaki1 Did this work without errors when you viewed one of these pages?

Reply:
Yes it did :)